spec/models/select_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 RSpec.describe Select, type: :model do
   subject { create(:select) }
-  LOCALIZATION_PREFIX = "activerecord.errors.models.select.attributes".freeze
+  LOCALIZATION_PREFIX = 'activerecord.errors.models.select.attributes'
```

Comment:
what is the purpose of this change?

Reply:
I ran rubocop for the project and it was corrected by it, I will fix it.

Reply:
Rubocop removed double quotes. Just add `freeze` again

Reply:
Thanks, done)